File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
      int n = nums.size();
      if(n<=2)
      {
        return n;
      }
      int start =1 ;
      for(int i=2 ; i<n ; i++)
      {
        if(nums[i] != nums[start-1])
        {
            start++;
            swap(nums[start], nums[i]);
        }
      }
      return start+1;
    }

};

File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        start = 1
        for i in range (2, len(nums)):
            if nums[i] != nums[start-1]:
                start +=1
            nums[start] = nums[i]
        
        return start+1
        